source/includes/code-examples/change-streams/change-streams.cs

```diff
@@ -65,6 +65,115 @@ await cursor.ForEachAsync(change =>
 }
 // end-change-stream-pipeline
 
+// start-split-event-helpers-sync
+// Fetches the next complete change stream event
+private static ChangeStreamDocument<TDocument> GetNextChangeStreamEvent<TDocument>(
```

Comment:
@BorisDog, can you take a look at the code examples? It seems you've worked on the implementation of this so you can better spot errors